.buildkite/commands/package_windows.ps1

```diff
@@ -0,0 +1,50 @@
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
+
+& "prepare_windows_host_for_node.ps1"
+
+# First try to get the env var from the process environment
+$windowsCertPassword = [System.Environment]::GetEnvironmentVariable('WINDOWS_CODE_SIGNING_CERT_PASSWORD', [System.EnvironmentVariableTarget]::Process)
+If ([string]::IsNullOrEmpty($windowsCertPassword)) {
+    # If it fails, try from the machine-wide environment
+    $windowsCertPassword = [System.Environment]::GetEnvironmentVariable('WINDOWS_CODE_SIGNING_CERT_PASSWORD', [System.EnvironmentVariableTarget]::Machine)
+}
+
+If ([string]::IsNullOrEmpty($windowsCertPassword)) {
+    Write-Host "[!] WINDOWS_CODE_SIGNING_CERT_PASSWORD is not set in either process or machine environments."
+    Exit 1
+} else {
+    [System.Environment]::SetEnvironmentVariable('CSC_KEY_PASSWORD', $windowsCertPassword, [System.EnvironmentVariableTarget]::Machine)
+    Write-Host "Environment variable CSC_KEY_PASSWORD set to the value of WINDOWS_CODE_SIGNING_CERT_PASSWORD."
+}
```

Comment:
I need to followup on whether `$env:<VAR NAME>` would be better here and save the trouble of doing multiple checks.